app/controllers/subscriptions_controller.rb

```diff
@@ -3,15 +3,16 @@ class SubscriptionsController < ApplicationController
   before_action :assign_attributes
 
   def new
-    @back_url = @frequency.blank? ? govuk_url : url_for(action: :new, topic_id: @topic_id)
```

Comment:
Out of scope for today/this PR - but the whole `govuk_url` thing looks a bit fishy and ripe for a rethink.

Reply:
I thought the same. I've got a follow-up item to think about it more.